qemu/tests/balloon_hotplug.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import time
+import random
+
+from virttest.qemu_devices import qdevices
+from virttest import error_context
+from qemu.tests import balloon_check
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test hotplug of balloon devices.
+
+    1) Boot up guest w/o balloon device.
+    2) Hoplug balloon device and check hotplug successfully or not.
+    3) Do memory balloon.
+    4) Unplug balloon device and check unplug successfully or not.
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+    def memory_balloon():
+        if params['os_type'] == 'windows':
+            balloon_test = balloon_check.BallooningTestWin(test, params, env)
+        else:
+            balloon_test = balloon_check.BallooningTestLinux(test, params, env)
+        for tag in params.objects('test_tags'):
+            params_tag = params.object_params(tag)
+            error_context.context("Running %s test" % tag, logging.info)
+            balloon_type = params_tag['balloon_type']
+            min_sz, max_sz = balloon_test.get_memory_boundary(balloon_type)
```

Comment:
(monitor qmp1)    actual: 1073741824
Sending command: cat /proc/meminfo |grep MemFree
list index out of range
Context: Running evict test
Sending command: cat /proc/meminfo |grep MemFree
list index out of range

need to raise error